wiki/Play_style/Double_tapping/id.md

```diff
@@ -0,0 +1,22 @@
+---
+tags:
+  - keyboard
+  - tap
+  - hybrid
+  - play style
+---
+
+# Double tapping
+
+*Berbeda konteks dengan [Alternating](/wiki/Play_style/Alternating).*
+
+**Double tapping** adalah gaya tapping untuk [osu!](/wiki/Game_mode/osu!) dimana pemain mengetuk circles, menggeser slider, dan memutar spinner dengan menekan kedua tombol pada keyboard mereka (`Z` dan `X` secara default). Gaya ini paling sering digunakan sebagai pengganti sementara gaya tapping [stacks](/wiki/Mapping_Techniques/Stack) dan [streams](/wiki/Beatmap/Pattern/Stream) yang cepat dan terlalu sulit untuk sebagian pemain. Gaya ini dikenal paling tidak akurat.
+
+## Penggunaan
+
+Double tapping biasanya menggantikan gaya tapping utama pemain karena mudah digunakan dan meningkatkan "kecepatan" (meskipun pemain sebenarnya tidak mengetuk lebih cepat). Cara untuk mendapatkan kecepatan yang lebih besar ini dapat dicapai dengan memanfaatkan hit window yang bertumpukan, muncul keseringan pada not-not bertumpuk dan stream. Jadi, satu input tombol terhitung sebagai satu hit circle dalam stack, lingkaran berikutnya juga akan tercatat sebagai hit karena hit window mereka sangat berdekatan sehingga saling tumpang tindih. Tetapi sebagai akibatnya, hampir tidak mungkin untuk mengontrol waktu tapping ini yang dapat mengurangi [akurasi](/wiki/Accuracy) pemain.
```

Comment:
```suggestion
Double tapping biasanya menggantikan gaya tapping utama pemain karena mudah digunakan dan meningkatkan "kecepatan" (meskipun pemain sebenarnya tidak mengetuk lebih cepat). Peningkatan kecepatan ini dicapai dengan mengambil keuntungan tumpang tindih dari hit window yang paling sering terjadi pada stack dan stream. Jadi, satu input tombol terhitung sebagai satu hit circle dalam stack, circle berikutnya juga akan terhitung sebagai hit karena hit window mereka sangat berdekatan sehingga saling tumpang tindih. Tetapi sebagai akibatnya, hampir tidak mungkin untuk mengontrol waktu tapping ini yang dapat mengurangi [akurasi](/wiki/Accuracy) pemain.
```

Reply:
missing several phases on EN
`This increase in speed is achieved by taking advantage of the overlapping hit windows that occur most frequently in stacks and streams.`